_posts/2016-01-05-raft-paxos.html

```diff
@@ -51,84 +51,84 @@
 	</div>
 </div></div>
 
 <div class="brim alt principles1"><div class="content">
 	<div class="principle edge">
 		<div class="title">Principle 2. Reduced read quorums.</div>
-		<div>When proposer receives majority $f+1$ of the 'promises' it may choose to ignore one 'promise' and it doesn't affect the consistency if the size of the cluster is even. I wrote about it and proved it in the <a class="link" href="http://rystsov.info/2015/12/30/read-write-quorums.html">Read write quorums in Paxos</a> post.</div>
+		<div>When a proposer receives majority of $f+1$ of the 'promises' it may choose to ignore one 'promise'. It doesn't affect consistency if the size of the cluster is even. I wrote about it and proved it in the <a class="link" href="http://rystsov.info/2015/12/30/read-write-quorums.html">Read write quorums in Paxos</a> post.</div>
 	</div>
 </div></div>
 
 <div class="brim"><div class="content">
-	<p>Let's review <a class="link" href="http://rystsov.info/2015/09/16/how-paxos-works.html">How Paxos works</a>, describe joint consensus, apply it to Single Decree Paxos and prove that the consistency of the system holds during the transition from $2f+1$ to $2f+2$ nodes cluster.</p>
+	<p>Let's review <a class="link" href="http://rystsov.info/2015/09/16/how-paxos-works.html">How Paxos works</a>, describe joint consensus, apply it to Single Decree Paxos and prove that the consistency of the system holds during the transition from a $2f+1$ to a $2f+2$ node cluster.</p>
 
 	<h4>How Paxos works</h4>
 
-	<p>When a proposer receives a request from a client to change the distributed variable in the $2f+1$ nodes paxos cluster it should:</p>
+	<p>When a proposer receives a request from a client to change the distributed variable in the $2f+1$ node paxos cluster it should:</p>
 
 	<ol>
-		<li>Ask acceptors to provide the current value (send 'prepare' message).</li>
-		<li>Wait until majority of the acceptors answer (at least $f+1$ nodes).</li>
-		<li>Select the value with max ballot number.</li>
+		<li>Ask acceptors to provide the current value (send the 'prepare' message).</li>
+		<li>Wait until majority of the acceptors (at least $f+1$ nodes) answer.</li>
+		<li>Select the value with the maximum ballot number.</li>
 		<li>Perform a transformation of the value.</li>
 		<li>Send the new value back to the acceptors.</li>
-		<li>Wait until majority of the acceptors answer (at least $f+1$ nodes).</li>
+		<li>Wait until a majority of the acceptors (at least $f+1$ nodes) answer.</li>
 		<li>Send confirmation to the client.</li>
 	</ol>
 
-	<p>To read a value proposer should use the identity function as the transformation.</p>
+	<p>To read a value, a proposer should use the identity function as the transformation.</p>
 
 	<h4>Joint consensus</h4>
 
-	<p>The idea behind joint consensus is</p>
+	<p>The idea behind joint consensus is:</p>
 
 	<ol class="joint1">
-		<li>To start work with old $\mathfrak{O}$ and new $\mathfrak{N}$ set of acceptors simultaneously.</li>
```

Comment:
Uhh, are you sure about the pick of symbols? Here's how they look on my machine:

<img width="972" alt="screen shot 2016-01-11 at 19 12 47" src="https://cloud.githubusercontent.com/assets/1493464/12240645/56d26936-b897-11e5-81bd-1d30349062fb.png">